src/python_inspector/resolve_cli.py

```diff
@@ -151,6 +151,11 @@ def print_version(ctx, param, value):
     help="Enable collection of requirements in setup.py that compute these"
     " dynamically. This is an insecure operation as it can run arbitrary code.",
 )
+@click.option(
+    "--prefer-source",
+    is_flag=True,
+    help="Prefer source distributions over binary distributions.",
```

Comment:
We need more documentation about this option behavior.
For example, what append when no source distribution is available?
This should be documented here, as well as in the code, in the docstring, in the changelog, and in the main documentation.

Reply:
Done!